server/service/src/sync/translations/master_list.rs

```diff
@@ -13,15 +15,17 @@ pub struct LegacyListMasterRow {
     note: String,
 }
 
+fn match_pull_table(sync_record: &SyncBufferRow) -> bool {
+    sync_record.table_name == LegacyTableName::LIST_MASTER
+}
```

Comment:
inline, its ok to repeat yourself :)

Reply:
It sure is ok, it's not ok if doing dry code leads to complexity, this is as simple as it gets